frontend/components/Documents.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from "react";
+// import * as PropTypes from "prop-types";
+// import STYLES from "./Documents.module.scss";
+
+const Documents = () => {
+
+    const [docData, setDocData] = useState(null);
+
+    useEffect(() => {
+        fetch("/api/all_documents")
+            .then(response => response.json())
+            .then(data => {
+                setDocData(data);
+            });
+    }, []);
+
+    return (
+        <div>
+            <h1>This is the Documents page.</h1>
+            <p>
+                This page displays all the documents stored in backend.
+            </p>
+            {
+                docData
+                    ? <div>Documents:
+                        {docData.map((doc, i) => {
+                            return (<>
+                                <p>Document {i}</p>
+                                <ul key={i}>
+                                    {Object.keys(doc).map((key, i) => {
+                                        return (
+                                            <li key={i}>{key}: {doc[key]}</li>
+                                        );
+                                    })}
+                                </ul>
+                            </>
+                            );
```

Comment:
I think the markup here could be slightly adjusted. As is stands, each `Document` instance is going to create a new `<ul>` element. While that's not necessarily inappropriate, it seems to me like a more conventional markup structure would be to have a wrapping `<ul>` element in which an `<li>` element surrounds the contents of each `Document` instance. Those contents can be rendered however you'd like, including with their own `<ul>` element.

As it currently stands, semantically, each document is being rendered as a `<p>` tag and a `<ul>` tag with no semantic relationship between the two and no semantic relationship between each other. Making them all `<li>` in an surrounding `<ul>` indicates that they are all related to one another in some capacity.